### ISSUE TITLE
[FR] - Turn Image to Latex and Delete

### ISSUE THREAD

**Issue by ndharari** [lucasvanmol/obsidian-latex-ocr#17]
Hi! I have this workflow with Reading Notes. For any math equation that I want to store in my notes when I read a PDF, I save it as an image and add it to my notes. Ideally, to avoid excess images, I would like to turn these images to LaTex. Previously, I did this with Mathpix, but this plugin is a great replacement. Now, to the point:

It would be great if we had an option of the like "replace this image with LaTex" that would delete the image file and replace it with the OCR'd code. Alternatively, the plugin could bring up a pop up saying "this is the result from the OCR, wanna replace the image and delete it, just replace, no delete or do nothing. 

TL;DR: Add an option to replace image by the latex output and delete the stored file.

**Comment by lucasvanmol** [comment]
Hey, thanks for the suggestion! Currently, there is a "Generate Latex" option that should pop up when you right-click an image or image link - although I agree this feature is not well documented. It doesn't, however, delete the image, instead just copies the latex to your clipboard.

Your pop up suggestion sounds like a good idea, which I'll look into, thanks!
